samples/exemplar/src/test/kotlin/com/squareup/exemplar/ActionsTestModule.kt

```diff
@@ -0,0 +1,20 @@
+package com.squareup.exemplar
+
+import com.google.inject.name.Names
+import misk.crypto.Cipher
+import misk.inject.KAbstractModule
+
+class ActionsTestModule : KAbstractModule() {
```

Comment:
fwiw in this case I'd say you could use constructor injection for the action class and then pass in the FakeCipher. IMO that'd be easier to follow

Reply:
Agree, I'll do that in another sample app I'll generate instead of `examplar`